docs/user-guide/project.md

```diff
@@ -28,120 +28,218 @@ A user menu with different actions is accessible for each member of the project
+Now you should see the new property in the box as seen below:
+![Layout after the creation of the property.](../assets/images/Fig17.png)
 
-![Data model editor | Step 4: Define property's label and comment and set the cardinality corresponding to the resource class.](../assets/images/data-model-property-create.png)*Data model editor | Step 4: Define property's label and comment and set the cardinality corresponding to the resource class.*
+### Define Lists
+One of the possible property types to choose from is *List*. Lists are very useful if you want to use controlled vocabulary to describe something. Typical examples are keywords. In your book example it may be useful to define a property which describes to which category of literature your pdf of a book belongs. **Before** you can add a property of type *List* to your data model, you have to define this list. For the definiton of a list you have to change to the `Lists` tab:
+![List tab.](../assets/images/Fig34.png)
 
-For each property in a resource class you can define some rules. For example you can define if one of the property can have `multiple` values or if a property is a mandatory (`requiered`) field.
+Click `Create your first list`. If there is already a list defined, click `New list`. A new window pops up where you have to enter a name for your list (`List label`) and a short description, then click `Create`.
+![Create new list.](../assets/images/Fig35.png)
 
-After adding more than one property to a resource class you can change the displayed order by drag'n'drop the properties in the list.
+As soon as the list is created you can continue with the definition of your data model. The single list items you can define later. How to do this will be explained below. We first focus on the definition of a list property in the data model. Currently there is only one option for the display of a property of type *List*, namely *Dropdown*. It is capable to display flat as well as hierarchical lists.
+![Define a property of type *List*.](../assets/images/Fig40.png)
 
-<!--- TODO: add definition for each property type and maybe also for the representation type
+A new window opens up and as in the case of other properties you have to add a label, a desciption and to choose whether multiple values are allowed and/or whether this property is a required field. But in addition you have to select the list which contains your controlled vocabulary.
+![Select list for list property.](../assets/images/Fig41.png)
 
-### Connect SOURCE TYPES in the data model (optional)
+#### How to define items in a list
+In our example we have created a list named *Category*. Now it is time to define the list items. We will as first hierarchy in our list some main literature genres. Enter the name of the list node and click the `+` as shown below:
```

Comment:
We will as first hierarchy in our list some main literature genres. - > We will list some main literature genres as the first hierarchy in our list.